dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -351,6 +351,14 @@ breeze build-docs --clean-build --package-filter apache-airflow-providers \
    --package-filter 'apache-airflow-providers-*'
 ```
 
+You can also use shorthand names as arguments instead of using the full names
+for airflow providers. Example:
+
+```shell script
+cd "${AIRFLOW_REPO_ROOT}"
+breeze build-docs cncf.kubernetes sftp --clean-build
+```
```

Comment:
No need to mention this here. This part of the doc is about building for all packages

Reply:
I see. Sure removing it